code/modules/antagonists/xeno/xeno.dm

```diff
@@ -0,0 +1,37 @@
+/datum/team/xeno
+	name = "Aliens"
+
+//Simply lists them.
+/datum/team/xeno/roundend_report()
+	var/list/parts = list()
+	parts += "<span class='header'>The aliens were:</span>"
+	parts += printplayerlist(members)
+	return "<div class='panel redborder'>[parts.Join("<br>")]</div>"
+
+/datum/antagonist/xeno
+	name = "Xenomorph"
+	job_rank = ROLE_ALIEN
+	show_in_antagpanel = FALSE
+	var/datum/team/xeno/xeno_team
+
+/datum/antagonist/xeno/create_team(datum/team/xeno/new_team)
+	if(!new_team)
+		for(var/datum/antagonist/xeno/X in GLOB.antagonists)
+			if(!X.owner || !X.xeno_team)
+				continue
+			xeno_team = X.xeno_team
+			return
+		xeno_team = new
+	else
+		if(!istype(new_team))
+			CRASH("Wrong xeno team type provided to create_team")
```

Comment:
What if some madman adds xeno ERT, or jolly xenomorph santa?

Reply:
This is problem with current team implementation in general, would have to think a bit how to make these less strict.